courses/free-courses-te.md

```diff
@@ -0,0 +1,205 @@
+# Index
+
+* [Android](#android)
+* [Angular](#angular)
+* [Artificial Intelligence](#artificial-intelligence)
+* [Automata Theory](#automata-theory)
+* [Bash and Shell](#bash-and-shell)
+* [C](#c)
+* [C++](#c++)
+* [Cloud Computing](#cloud-computing)
+* [Compiler Design](#compiler-design)
+* [CSS](#css) 
+* [Database Management Systems](#database-management-systems)
+* [Data Science](#data-science)
+* [Data Structures](#data-structures)
+* [Deep Learning](#deep-learning)
+* [Ethical Hacking](#ethical-hacking)
+* [ExpressJS](#expressjs)
+* [HTML](#html)
+* [Java](#java)
+* [Javascript](#javascript)
+* [Laravel](#laravel)
+* [Machine Learning](#machine-learning)
+* [MongoDB](#mongodb)
+* [NextJS](#nextjs)
+* [NodeJS](#nodejs)
+* [PHP](#php)
+* [Python](#python)
+* [ReactJS](#reactjs)
+* [SQL](#sql)
+* [VueJs](#vuejs)
+
```

Comment:
Add another new line here and you are good to go

Reply:
Yeah, I just did that. Thank you so much! I did not know that there has to be uniformity in the blank lines that I had left. Actually, it kinda feels demotivating that the number of blank lines matters.

Reply:
Hmm, actually it's in the contributing.md file. Always look into the contributing and code of conduct before contributing to open source.

Reply:
Oh, I did not know that. Thanks for the tip will definitely follow it 👍